<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="4" total="5" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Failed to EnterSophie_Auto_328 intxtShortDesc]]>
    </line>
    <line>
      <![CDATA[Failed to EnterCreated as part of automation intxtVersionNotes]]>
    </line>
    <line>
      <![CDATA[Failed to Click in imgBtnNext]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="445096" started-at="2019-05-29T13:23:46Z" finished-at="2019-05-29T13:31:11Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="445096" started-at="2019-05-29T13:23:46Z" finished-at="2019-05-29T13:31:11Z">
      <class name="regressionTests.RegressionCases">
        <test-method status="PASS" signature="setUp()[pri:0, instance:regressionTests.RegressionCases@1fc2b765]" name="setUp" is-config="true" duration-ms="83" started-at="2019-05-29T13:23:46Z" finished-at="2019-05-29T13:23:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="Verify_DownloadedCSV_For_Existing_Version()[pri:1, instance:regressionTests.RegressionCases@1fc2b765]" name="Verify_DownloadedCSV_For_Existing_Version" duration-ms="444961" started-at="2019-05-29T13:23:46Z" finished-at="2019-05-29T13:31:11Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at driver.ActionClass.checkboxSelect(ActionClass.java:818)
at driver.DriverEngine.mainMethod(DriverEngine.java:339)
at regressionTests.RegressionCases.Verify_DownloadedCSV_For_Existing_Version(RegressionCases.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Failed to EnterSophie_Auto_328 intxtShortDesc]]>
            </line>
            <line>
              <![CDATA[Failed to EnterCreated as part of automation intxtVersionNotes]]>
            </line>
            <line>
              <![CDATA[Failed to Click in imgBtnNext]]>
            </line>
          </reporter-output>
        </test-method> <!-- Verify_DownloadedCSV_For_Existing_Version -->
      </class> <!-- regressionTests.RegressionCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
